crates/rolldown/tests/esbuild/ts/ts_enum_jsx/artifacts.snap

```diff
@@ -61,11 +60,10 @@ let x;
 (function(_x) {
 	let y;
 	(function(_y) {
-		let React$1 = /* @__PURE__ */ function(React$2) {
-			React$2["Fragment"] = "div";
-			return React$2;
+		_y.React = /* @__PURE__ */ function(React$1) {
```

Comment:
Such inline breaks the behavior of inferring`.name` property of nameless function. Does the `keepNames` handle this case? @IWANABETHATGUY cc

Reply:
This case is the only one that seems to break the runtime behavior, but @sapphi-red's solution looks good to me